python/pfs/drp/stella/fitFluxCal.py

```diff
@@ -342,7 +345,11 @@ def __init__(
         self.arms: dict[int, str] = getExistentArms(pfsMerged)
 
         self.bbFlux: dict[int, list[tuple[float, float, str]]] = {
-            fiberId: list(zip(bbFlux, bbFluxErr, filterNames))
+            fiberId: [
+                (flux, err, filter)
+                for flux, err, filter in zip(bbFlux, bbFluxErr, filterNames)
+                if filter not in unusedBroadbandFilters
```

Comment:
Probably should cast `unusedBroadbandFilters` to a `set` for efficiency.

Reply:
I was not sure whether or not `set` outperforms `list` when the number of elements is only a few, but experiment shows `set` is indeed faster when there are only two elements.